lib/metadata/BucketInfo.js

```diff
@@ -50,6 +54,7 @@ export default class BucketInfo {
         this._mdBucketModelVersion = mdBucketModelVersion || 0;
         this._transient = transient || false;
         this._deleted = deleted || false;
+        this._serverSideEncryption = serverSideEncryption || null;
```

Comment:
We should be updated the mdBucketModelVersion because of this addition.  From this, I realized our update system was not quite ideal so I opened this https://github.com/scality/S3/pull/114